README.md

```diff
@@ -1 +1,24 @@
-# llm_validation
\ No newline at end of file
+## HOW TO RUN:
+### 1. Create constants.py file with:
+- PROMPT
+- OPTIONS
+- INSTRUCTION
+- INPUT_TEXT_1
+- INPUT_TEXT_2
+- ...
+- INPUT_TEXT_9
```

Comment:
those variables are not constants, we want to be able to change it, so I would use .config file for PROMPT, OPTIONS, and INSTRUCTION. This config file should be passed through UI. 

Regarding Input text, it is data set, so just read it from CSV file.

Reply:
Ok  now will change. Actually in program we only read input_text (specification) from csv. I just added oppotrunity to write new testcase to csv by write_to_csv function, If we have specification we just add new input_text_x and add it to csv file by script. Should I only remove that from readme or from constants.py also? I mentioned in this file we have only default options, instruction and prompt, if user provide another in UI then it will use user's data not default

Reply:
whats the purpose of this for the end user?